tutorials/migrating/upgrading_to_godot_4.3.rst

```diff
@@ -0,0 +1,350 @@
+**WebXRInterface**
+Method ``get_input_source_tracker`` changes return type from ``XRPositionalTracker`` to ``XRControllerTracker``           |✔️|                 |❌|                  |✔️|                  `GH-90645`_
+**XRServer**
+Method ``get_tracker`` changes return type from ``XRPositionalTracker`` to ``XRTracker``                                  |✔️|                 |❌|                  |❌|                  `GH-90645`_
+========================================================================================================================  ===================  ====================  ====================  ===========
+
+Editor plugins
+^^^^^^^^^^^^^^
+
+========================================================================================================================  ===================  ====================  ====================  ===========
+Change                                                                                                                    GDScript Compatible  C# Binary Compatible  C# Source Compatible  Introduced
+========================================================================================================================  ===================  ====================  ====================  ===========
+**EditorInspectorPlugin**
+Method ``add_property_editor`` addsa a new ``label`` optional parameter                                                   |✔️|                 |✔️ with compat|      |✔️|                  `GH-92322`_
+**EditorPlugin**
+Method ``add_control_to_bottom_panel`` adds a new ``shortcut`` optional parameter                                         |✔️|                 |✔️ with compat|      |✔️|                  `GH-88081`_
+Method ``add_control_to_dock`` adds a new ``shortcut`` optional parameter                                                 |✔️|                 |✔️ with compat|      |✔️|                  `GH-88081`_
+**EditorSceneFormatImporterFBX**
+Type renamed to ``EditorSceneFormatImporterFBX2GLTF``                                                                     |❌|                 |❌|                  |❌|                  `GH-81746`_
+========================================================================================================================  ===================  ====================  ====================  ===========
+
+Behavior changes
+----------------
+
+In 4.3 some behavior changes have been introduced, which might require you to adjust your project.
+
+Core
+^^^^
+
+.. note::
+
+    Binary serialization was modified to fix some issues with the serialization of scripted Objects and typed Arrays (`GH-78219`_).
+    This breaks compat with script encoding/decoding.
+
+.. note::
+
+    ``PackedByteArray`` is now encoded as a base64 string, this breaks compatibility in resources that contain them
+    making them no longer usable in older versions of the engine (`GH-89186`_).
```

Comment:
This may be worth noting when the older versions are patched, but https://github.com/godotengine/godot/pull/91250 should add support for decoding the **PackedByteArray** string even in 4.2 and 4.1.

Reply:
My English is not very good, but the basic idea is like this:

> `PackedByteArray` is now able to use a more compact base64 encoding for storage. But the trade-off is that it breaks compatibility, meaning that older versions of Godot may not be able to open resources saved by 4.3.
>
> To maximize compatibility, this new storage format will only be enabled for resources and scenes that contain large `PackedByteArray`s for now. Support for this new format will also be added in patch updates for older versions of Godot. Once all supported Godot versions are able to read the new format, we will gradually retire the compatibility measures and have all resources and scenes use the new storage format.